public_html/js/index.js

```diff
@@ -4,12 +4,20 @@ $( document ).ready( function () {
 		selector: '[data-toggle="tooltip"]'
 	} );
 	$( '.records' ).on( 'click', '.js-save-state', function () {
-		saveState( $( this ).data( 'id' ), $( this ).data( 'status' ) );
+		var status = $( this ).data( 'status' ),
+			id = $( this ).data( 'id' );
+
+		// undo review if they click on the button with the same status as the record
+		if ( status === $( '.record-' + id ).data( 'status' ) ) {
+			undoReview( id, status );
+		} else {
+			saveState( id, status );
+		}
 	} );
 	$( '.records' ).on( 'click', '.js-compare-button', function () {
 		// pass the dataset of the element as an object to toggleComparePane
 		toggleComparePane.call(this, this.dataset);
-	});
+	} );
```

Comment:
}) is the standard when doing MW development. I didn't find anything in the Manual for coding conventions but you'll find examples in the code. I just rely on Bryan's code to check myself: https://github.com/wikimedia/wikimedia-iegreview/blob/master/public/js/site.js#L65